test/blah.test.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import * as ReactDOM from 'react-dom';
-import { Default as Thing } from '../stories/Thing.stories';
+import { Default as Button } from '../stories/Button.stories';
 
-describe('Thing', () => {
+describe('Button', () => {
   it('renders without crashing', () => {
     const div = document.createElement('div');
-    ReactDOM.render(<Thing />, div);
+    ReactDOM.render(<Button />, div);
```

Comment:
Tests are currently not passing with the following error:
`SyntaxError: /liane-ui/test/blah.test.tsx: Support for the experimental syntax 'jsx' isn't currently enabled (8:21):`

The current test is also not asserting any result from the `ReactDOM.render`. I recommend we use **jest** combined with **enzyme** to assert react and dom rendering.

The file name `blah.test.tsx` also does not reflect the test for the Button component. I recommend we use the following directory structure:

```
├── src
│   ├── components
│   │   └── Button
│   │       │   └── test
│   │       │       └── index.tsx
│   │       └── index.tsx
│   ├── index.css
│   └── index.tsx
```

This structure incorporates the modularized aspect of the components and makes it easier to grow and diagnose issues related to a single component.